public/app/features/alerting/unified/components/admin/ExternalAlertmanagers.tsx

```diff
@@ -121,10 +136,43 @@ export const ExternalAlertmanagers = () => {
   };
 
   const noAlertmanagers = externalAlertManagers?.length === 0;
+  const noDsAlertmanagers = externalDsAlertManagers?.length === 0;
+  const hasExternalAlertmanagers = !(noAlertmanagers && noDsAlertmanagers);
 
   return (
     <div>
       <h4>External Alertmanagers</h4>
+
+      <h5>Alert managers data sources</h5>
+      <div className={styles.muted}>
+        During the Alert manager data source configuration you can choose to send Grafana managed alerts to that Alert
+        manager. <br />
+        Here you can see the list of all Alert manager data sources which receive Grafana managed alerts.
+      </div>
+      <div className={styles.externalDs}>
+        {externalDsAlertManagers.map((ds) => (
+          <Card key={ds.uid}>
```

Comment:
Can we make this a link back to the datasource? It would be useful to be able to link back.